src/ls/ls.rs

```diff
@@ -611,7 +613,14 @@ fn display_file_name(
     }
     let mut width = UnicodeWidthStr::width(&*name);
 
-    let color = options.opt_present("color");
+    let color = match options.opt_str("color") {
+        None => true,
+        Some(val) => match val.as_ref() {
+            "always" => true,
+            "auto" => stdout_isatty(),
+            "never" | _ => false,
+        },
```

Comment:
When GNU `ls` receives an invalid colour argument, it outputs the following:
```
[arni][~]% ls --color=abcdefghi
ls: invalid argument ‘abcdefghi’ for ‘--color’
Valid arguments are:
  - ‘always’, ‘yes’, ‘force’
  - ‘never’, ‘no’, ‘none’
  - ‘auto’, ‘tty’, ‘if-tty’
Try 'ls --help' for more information.
```
These aliases don't appear in the manpages, but I think we should support them nonetheless.

Reply:
Thanks for the review.
Good idea, I had not seen those. I have added support for those aliases.